test/box-luatest/gh_10057_user_role_test.lua

```diff
@@ -0,0 +1,37 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new()
+    cg.server:start()
+end)
+
+g.after_all(function(cg)
+    cg.server:drop()
+end)
+
+g.test_origins_in_privs = function(cg)
+    cg.server:exec(function()
+        box.schema.user.create('testuser')
+        box.schema.role.create('manager')
+        box.schema.role.create('director')
+        box.schema.user.grant('testuser', 'manager')
```

Comment:
Let's check a transition in another test: role is granted to another role that is granted to a user.